queries.sql

```diff
@@ -31,4 +31,52 @@ WHERE NAME <> 'Gabumon';
 
 SELECT *
 FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3;  
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3;  
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals
+
+UPDATE animals
+SET species = 'pokemon';
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN
+DELETE
+FROM animals;
+ROLLBACK;
+
```

Comment:
- [ ] Please add a select statement after the rollback to verify the change was rollback.

Reply:
resolved 👍